dashboard/src/components/access/RoleManager.vue

```diff
@@ -167,12 +171,13 @@ limitations under the License.
       },
       addUserRole() {
         this.isSaving = true;
+        this.table.options.busy = true;
         const pkiAuthenticated = this.$store.getters.isPkiAuthenticated;
 
         AccessService.saveUserRole(this.project.projectId, this.selectedUser, this.role, pkiAuthenticated)
-          .then((userInfo) => {
-            this.userAdded(userInfo);
+          .then(() => {
             this.$emit('role-added', { userId: this.selectedUser.userId, role: this.role });
+            this.loadData();
```

Comment:
@sudo-may - I think we need to call `loadData()` in the `deleteUserRole()` method as well

Reply:
@sudo-may - nvm, I see you handled removing a root user from the supervisors table in the delete in `SecuritySettings.vue`